pkgs/development/compilers/gcc/builder.sh

```diff
@@ -283,19 +291,14 @@ postInstall() {
         fi
     done
 
+    # TODO: on musl *-c++* fails to match
     for i in $out/bin/c++ $out/bin/*-c++* $out/bin/*-g++*; do
         if cmp -s $out/bin/g++ $i; then
             ln -sfn g++ $i
         fi
     done
-
-    # Two identical man pages are shipped (moving and compressing is done later)
-    for i in "$out"/share/man/man1/*g++.1; do
-        if test -e "$i"; then
-            man_prefix=`echo "$i" | sed "s,.*/\(.*\)g++.1,\1,"`
-            ln -sf "$man_prefix"gcc.1 "$i"
-        fi
-    done
```

Comment:
Are you sure this can be safely removed? It seems eed46887ca8b4b4f0fe473e32e042c0ee3a6eee5 had a good reason for that.

Cc: @iamthememory

Reply:
Thank you for finding the commit. This might need a nullglob then.

Reply:
I've hacked it away, but I'm considering just disabling failglob and adding a fixme comment. Not sure which is better.